packages/create-remix/index.ts

```diff
@@ -741,6 +750,62 @@ async function updatePackageJSON(ctx: Context) {
     JSON.stringify(sortPackageJSON(packageJSON), null, 2),
     "utf-8"
   );
+  return true;
+}
+
+async function updateDenoJSON(ctx: Context) {
```

Comment:
this only supports a `deno.json` file in a template without comments for simplicity's sake